src/HW_models/NRF_RADIO.c

```diff
@@ -654,6 +654,7 @@ void nrf_radio_timer_triggered(){
   } else if ( radio_state == RXDISABLE ){
     radio_state = DISABLED;
     Timer_RADIO = TIME_NEVER;
+    NRF_RADIO_regs.STATE = DISABLED;
```

Comment:
I would move this up to be under `radio_state` such that it matches the order used other places in this function.

@aescolar If the assignment to `NRF_RADIO_regs.STATE` is missing here, isn't it then also missing in L650?

Reply:
@thoh-ot it seems to be have been a silly omission in both places yes.
@cvinayak I guess you agree that this should have been set?
@jrotkiewicz would you be so kind as to correct it in both places and move it up one line as requested by @thoh-ot ?
With that I think we can merge it